src/OpenColorIO/OCIOYaml.cpp

```diff
@@ -270,6 +271,29 @@ inline void throwValueError(const std::string & nodeName,
     throw Exception(os.str().c_str());
 }
 
+// Duplicate Checker
+
+inline void checkDuplicates(const YAML::Node& node)
```

Comment:
The global methods must start with an upper case letter (i.e. we did not revisit all the existing files to conform them to the coding style) and add a space between `YAML::Node` and `&`.

Reply:
I had a confusion regarding this too. I tried to visited all the functions on `OCIOYaml.cpp` to check for both the capitalization and the space before the '&' symbol. Some functions had the space, others didn't. So I made a guess.

Reply:
That's the coding style but not all files are following it. So, only modifying the code you changed is enough.